libs/logic-apps-shared/src/utils/src/lib/models/template.ts

```diff
@@ -1,12 +1,12 @@
-export type SkuType = 'standard' | 'consumption';
-export type Kind = 'stateful' | 'stateless';
+const SkuType = ['standard', 'consumption'];
+export type SkuType = (typeof SkuType)[keyof typeof SkuType];
```

Comment:
Shouldn't we have similar for kinds?

Reply:
Discussing offline

Reply:
I left kind as string because it was causing type issues in many places; would this be something we want to enforce and if so how would we approach this?
![331d6766-6b45-44a7-b61e-c49df1dc81b2](https://github.com/Azure/LogicAppsUX/assets/144840522/5d5e2183-7e0d-46bd-bd40-10c8cce7f13b)